### DISCUSSION TITLE
[Bug] Speech bubbles linger after hover off

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#44155] [Other Feature Feedback, Questions, & Ideas]
The speech bubble pop-ups that appear when hovering over links in contribution activity & overview (and probably elsewhere) often linger after hovering off until a 2nd hover-on occurs. Instead, they should always instantly disappear upon hovering off
![hover-glitch](https://user-images.githubusercontent.com/10906554/212190321-36c6a39c-bbeb-4b6b-b578-6022f99b7d75.gif)

**Comment by radar-blip** [comment]
This was happening to me too, but seems to have stopped now.

**Reply by adamlui** [reply]
Interesting, did it stop after 10 hours ago?

**Reply by radar-blip** [reply]
Actually, I thought it stopped just before commenting (8 hours ago) but it seems to be happening again. It seems random.

**Reply by radar-blip** [reply]
And now, it is not happening, again. 😕

**Reply by radar-blip** [reply]
Seems to fix itself if you reload/refresh the page, but will happen again soon. 😃

**Comment by crystal-williams-brown** [comment]
This happens to me all the time and it makes it very difficult to review pull requests. Often the bubbles/popups will cover what I'm reading, a button I need or where I'm writing. The worst part is, I don't hover on purpose so they aren't helping me at all. On a day to day I know who the people are and what the links go to so I don't need hover information. So now I either play a game of move the bubble by purposefully hovering, reload the page when I mistakenly hover, or try to move extra carefully so I don't accidentally go near any triggers.

**Comment by stevehanson** [comment]
This issue seems to affect Firefox on a Mac and is reproducible. Reproduced in 1110.0b3 (developer version). A couple coworkers and myself have tried reproducing in Mac Safari and Mac Chrome and are not experiencing the problem there.

**Reply by adamlui** [reply]
It affected Firefox on my windows PC also

**Comment by radar-blip** [comment]
According to this discussion https://github.com/orgs/community/discussions/44630#discussioncomment-4729878
the team is aware of the problem and working on a fix.
